client/app/assets/styles/application.scss

```diff
@@ -284,16 +284,16 @@ textarea {
     border: $border-default;
     font-size: $font-size-16;
     line-height: $line-height-24;
-    background-color: $color-white;
+    background-color: var(--callout-background-color);
     display: inline-block;
 }
 
 .callout--success {
-    border-left: 3px solid $color-success;
+    border-left: 3px solid var(--callout-success-border-color);
 }
 
 .callout--warning {
-    border-left: 3px solid $color-warning;
+    border-left: 3px solid var(--callout-warning-border-color);
```

Comment:
I wouldn't make callout colors configurable since they are used to show success or warning states, so the green/orange colors have specific meaning and are relevant to the UX here.

Reply:
I changed it because a user mentioned it https://github.com/standardnotes/listed/issues/151#issuecomment-816599799, I will do the change specific to the "Thank" section which is where it's being used, does that sound ok?

Reply:
Since colors are conveying meaning here, I think it's important we keep this meaning consistent across the whole platform and make sure users are always given predictable feedback for their actions. @mobitar thoughts on this?

Reply:
This sounds like the correct theoretical approach but may cause problems if a user chooses to customize their blog in such a way that the warning/error colors become completely invisible against the background. Let's say someone chooses a red color to be their bg 💀 So I suppose allow them to customize this does give them more power, but we'll hope they'll use it correctly.

Reply:
Hm yeah, that's a really good point actually! We've been going over this with @daianaszwimer on Twist so I think we're all on the same page 👍